src/index.js

```diff
@@ -80,6 +81,7 @@ export default class Spinner extends React.Component {
 
   static defaultProps = {
     visible: false,
+    cancelable: true,
```

Comment:
@gsavvid you are right. Can you please only update the README.md with this new `prop`?